pages/en/contrib/Reporting-issues.md

```diff
@@ -41,6 +41,10 @@ There are three types of issues:
 
 ### Security issues
 
+We take security vulnerabilities in LoopBack very seriously for our users.
+
+It is recommended to never turn on debug logs on production that may contain sensitive data sent by the client. For example, it is ok to debug-log the payload of `PersistedModel.create` if debug logs are used only at development time.
```

Comment:
Suggest rewording:

In production, do not enable debug logs that may contain sensitive data; for example, the payload of `PersistedModel.create` should not be exposed in production.  Logging this type of information is acceptable in development, but not production.